packages/mongodb-memory-server-core/package.json

```diff
@@ -60,6 +60,7 @@
     "semver": "^7.3.2",
     "tar-stream": "^2.1.4",
     "tmp": "^0.2.1",
+    "tslib": "^2.0.1",
```

Comment:
We should try to keep `dependencies` as small as possible. I think that `importHelpers` with `tslib` are redundant even they help to reduce bundle size. 

`mongodb-memory-server` works on the backend, so bundle size is less critical than the number of prod dependencies.

Reply:
from my knowledge, many projects use this when they also use typescript - so if i understand the chain correctly, tslib will only be installed once (if the versions match up)

-> this in general help keeping the node_modules folder smaller on disk (you have probably seen these jokes about it already)